src/components/imageoptionseditor/imageoptionseditor.js

```diff
@@ -1,77 +1,112 @@
-define(["globalize", "dom", "dialogHelper", "emby-checkbox", "emby-select", "emby-input"], function(globalize, dom, dialogHelper) {
+define(["globalize", "dom", "dialogHelper", "emby-checkbox", "emby-select", "emby-input"], function (globalize, dom, dialogHelper) {
     "use strict";
 
     function getDefaultImageConfig(itemType, type) {
         return {
             Type: type,
             MinWidth: 0,
             Limit: "Primary" === type ? 1 : 0
-        }
+        };
     }
 
     function findImageOptions(imageOptions, type) {
-        return imageOptions.filter(function(i) {
-            return i.Type == type
-        })[0]
+        return imageOptions.filter(function (i) {
+            return i.Type == type;
+        })[0];
     }
 
     function getImageConfig(options, availableOptions, imageType, itemType) {
-        return findImageOptions(options.ImageOptions || [], imageType) || findImageOptions(availableOptions.DefaultImageOptions || [], imageType) || getDefaultImageConfig(itemType, imageType)
+        return findImageOptions(options.ImageOptions || [], imageType) || findImageOptions(availableOptions.DefaultImageOptions || [], imageType) || getDefaultImageConfig(itemType, imageType);
```

Comment:
can this be somehow line-wrapped?

Reply:
no need to wrapped here

Reply:
this line is very long, which makes it hard to read... but okay, let's leave it be.